### DISCUSSION TITLE
Istio port-natting 8443

### DISCUSSION THREAD

**Discussion by nikhil-ctc** [istio/istio#52180] [Q&A]
We have Istio set up with helm and has service entries for all external endpoints that our pods connect to which are all on port 443. Lately, we got a new external endpoint that communicates only on port 8443, so we have set new service entry with port number as 8443 but in our firewall we see the communication is happening on port 443, we suspect as port 8443 is part of Istio ports its port-natting

Trying to figure out how to mitigate this issue,
Thanks all in advance

**Comment by abasitt** [comment]
can you share your se?
